namada/namada-housefire.json

```diff
@@ -0,0 +1,62 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "namada",
+  "status": "upcoming",
+  "network_type": "testnet",
+  "website": "https://namada.net",
+  "pretty_name": "Namada",
+  "chain_type": "namada",
+  "chain_id": "housefire-cotton.d3c912fee7462",
+  "slip44": 118,
+  "bech32_prefix": "nam",
+  "daemon_name": "nam",
+  "key_algos": ["secp256k1"],
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "namnam",
+        "fixed_min_gas_price": 0.0025,
+        "low_gas_price": 0.0025,
+        "average_gas_price": 0.025,
+        "high_gas_price": 0.04
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "nam"
+      }
+    ],
+    "lock_duration": {
+      "time": "1209600s"
+    }
+  },
+  "codebase": {
+    "git_repo": "https://github.com/anoma/namada",
+    "language": {
+      "type": "rust",
+      "version": "1.21.4"
```

Comment:
0.44.1

Reply:
I think this states the Rust version. Which one should I use?

Reply:
I've removed this entry for now